packages/opentelemetry-test-utils/src/test-utils.ts

```diff
@@ -41,7 +41,6 @@ const dockerRunCmds = {
     'docker run --rm -d --name otel-mysql -p 33306:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=test_db -e MYSQL_USER=otel -e MYSQL_PASSWORD=secret mysql:5.7 --log_output=TABLE --general_log=ON',
   postgres:
     'docker run --rm -d --name otel-postgres -p 54320:5432 -e POSTGRES_PASSWORD=postgres postgres:16-alpine',
-  redis: 'docker run --rm -d --name otel-redis -p 63790:6379 redis:alpine',
```

Comment:
curious, why this is no longer needed. Is not used anymore?

Reply:
Correct, it is no longer used. It was used by `testUtils.startDocker('redis')` calls, but those have been removed in the commits so far to this PR.

When all packages' testing is switched over to using this new test-services stuff, then `startDocker`, `cleanUpDocker` and all those `dockerRunCmds` entries can be removed from this file.